src/hooks/useSubsetDataByNullifier.ts

```diff
@@ -24,6 +24,15 @@ export function useSubsetDataByNullifier() {
       }
       // requestPolicy: 'cache-and-network'
     });
+    
+  useEffect(() => {
+    executeSubsetDatasQuery({
+      contractAddress: contractAddress.toLowerCase(),
+      nullifier: recentWithdrawal.nullifier.toLowerCase(),
+      subsetRoot: recentWithdrawal.subsetRoot.toLowerCase()
+    });
+
+  }, [recentWithdrawal, contractAddress]);
```

Comment:
I'm not sure this is actually running when recentWithdrawal updates, so I'm just being explicit here to force the query. This was part of the debugging to really make sure I was returning data.